src/index.html

```diff
@@ -26,54 +26,42 @@
         <div class="frontpage-slogan-with-footnotes">
           <div class="frontpage-slogan">
             <h3>
-              Dart is an
-              <span data-toggle="popover" data-content="Dart is focused on client-side programming, be it in the browser or on mobile.">application</span> programming
-              <span data-toggle="popover" data-content="But it’s not just a language. Dart comes with well-crafted core libraries, a package manager, static analysis tools, and more.">language</span> that’s
-              <span data-toggle="popover" data-content="The syntax feels familiar, and the semantics are clean and consistent. No nasty surprises.">easy to learn</span>,
-              <span data-toggle="popover" data-content="Google alone has over 2 million lines of production Dart code. Apps can reach hundreds of thousands of lines of code.">easy to scale</span>, and deployable
-              <span data-toggle="popover" data-content="Dart runs fast in every modern browser, on the command line, on servers, and on mobile.">everywhere</span>.
-            </h3>
-            <h3>
-              Google depends on Dart to make
-              <span data-toggle="popover" data-content="For example, the next generation of AdWords (Google’s foremost money-making app) runs on Dart.">very large apps</span>.
+              Dart helps you craft beautiful, high-quality experiences
+              across all screens, with:
+
+              <ul>
+                <li> A <span data-toggle="popover" data-content="Dart started out being optimized for web apps, and is evolving to provide great support for mobile apps. Dart also runs on the command line and server-side.">client-optimized</span> language </li>
+                <li> Rich, powerful <span data-toggle="popover" data-content="For mobile apps, we recommend <a href='https://flutter.io'>Flutter</a>. For desktop apps, <a href='https://webdev.dartlang.org/angular'>AngularDart</a>.">frameworks</span> </li>
+                <li> Delightful, flexible <span data-toggle="popover" data-content="These range from <a href='/tools'>general-purpose tools</a> to framework-specific ones like the <a href='https://flutter.io/getting-started'><code>flutter</code> tool</a>.">tooling</span>
+                </li>
+              </ul>
+              <p><a href="/guides/get-started" class="frontpage-slogan__get-started btn btn-default btn-lg">Get Started</a> </p>
             </h3>
-            <p><a href="/guides/get-started" class="frontpage-slogan__get-started btn btn-default btn-lg">Get Started</a> <a href="/install" class="btn btn-lg">Install Dart</a></p>
+            <p> <em>Click <span data-toggle="popover" data-content="Try clicking dotted underlines in the text above and in the code sample.">underlined text</span> to learn more.</em> </p>
```

Comment:
I almost feel like we should get rid of this line now with the new popup.

Reply:
When the feature was on mouse-over, I think we could've deleted it, but now that you have to click, I think we need a hint. (I suppose we could test this supposition, somehow.)